src/algorithm/mod.rs

```diff
@@ -9,33 +9,15 @@ mod minimax;
 mod transposition_table;
 mod negamax;
 
-static PATTERNS: [((u8, u8, bool), isize, isize); 12] = [
```

Comment:
did you just removed this, or was it moved to another file ?

Reply:
Yep, it's gone, but no one was using it (at least on develop).